upgrading/topics/install_new_version.adoc

```diff
@@ -65,22 +60,43 @@ For Red Hat Enterprise Linux 8:
 +
  subscription-manager repos --enable=rh-sso-7.5-for-rhel-8-x86_64-rpms
 +
-[NOTE]
-====
-To disable older product repositories for both JBOSS EAP and {project_name} use:
 
- subscription-manager repos --disable=<OLDER_PRODUCT_REPO>
+. Disable older product repositories for both JBOSS EAP and {project_name}:
 
-To check the repositories use:
+ subscription-manager repos --disable=jb-eap-7.3-for-rhel-8-x86_64-rpms
+ subscription-manager repos --disable=rh-sso-7.4-for-rhel-8-x86_64-rpms
 
- yum repolist
-====
-. The RPM upgrade process will not replace any of your modified configuration files, and will instead create .rpmnew files for the default configuration of the new {project_name} version.
+. Check the list of repositories:
+
+ dnf repolist
+
+ Updating Subscription Management repositories.
+ repo id repo name
+ rh-sso-7.5-for-rhel-8-x86_64-rpms Single Sign-On 7.5 for RHEL 8 x86_64 (RPMs)
+ rhel-8-for-x86_64-appstream-rpms Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+ rhel-8-for-x86_64-baseos-rpms Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+
+. Back up any modified configuration files and custom modules.
+
+. Remove packages for RH-SSO 7.4.
++
+Normally the old packages are removed by `dnf upgrade`. However, the RH-SSO 7.5 packages use a format that conflicts with files handled by the JBoss EAP 7.3 package. Therefore, the RH-SSO 7.4 and JBoss EAP 7.3 packages must be removed.
```

Comment:
@andymunro Maybe add `manually` at the end of the last sentence? IOW, it to read like `Therefore, the RH-SSO 7.4 and JBoss EAP 7.3 packages must be removed manually.`